src/App.css

```
@@ -215,23 +407,58 @@
+    right: 15%;
+    bottom: -7vw;
+    z-index: 10;
   }
 }
 
 /* Tablet styles (min-width: 769px and max-width: 1024px) */
 @media (min-width: 769px) and (max-width: 1024px) {
-  .body-root-sets-each {
-    /* Add styles specific to tablet screens here */
-  }
 }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)